packages/invoices/src/OutboundShipment/api/hooks.ts

```diff
@@ -349,3 +350,36 @@ export const useDeleteSelectedLines = (): {
 
   return { onDelete };
 };
+
+export const useUpdateOutboundTax = () => {
```

Comment:
just noticed that line `208` should be
```    const stockLines = lines.nodes.filter(isA.stockOutLine);```

currently the outboundItems are being filtered by `isA.stockInLine` - sorry, must have missed that one!

Reply:
another issue - this is filtering out placeholder lines and I'd like to see placeholders..

Reply:
😢 I'm a silly goose sorry - fixed it for the non grouped version but not the grouped! 🐦  (OK close enough)